### ISSUE TITLE
using media types for errors

### ISSUE THREAD

**Issue by SteveLasker** [ietf-scitt/draft-birkholz-scitt-scrapi#4]
See [using media types for errors](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/62) from https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/

**Comment by aj-stein-nist** [comment]
I believe the salient question was from @OR13 was from the previous issue comment in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/62#issue-1641116305 was not answered.

> [](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/60#discussion_r1148486103)
> 
> Why?
> 
> Are there any references for this design choice I can review?

Answer: [RFC 9457 Section 3](https://datatracker.ietf.org/doc/html/rfc9457#section-3). However, it doesn't have MUST or other requirements even if it is a RFC, so not sure the media type structure is mandatory.

> The canonical model for problem details is a JSON [[JSON](https://datatracker.ietf.org/doc/html/rfc8259)] object. When serialized in a JSON document, that format is identified with the "application/problem+json" media type.

**Comment by SteveLasker** [comment]
Hi, @aj-stein-nist
We just moved this to SCRAPI to focus the discussion under this repo as we're splitting out the APIs to it's own draft, from the Architecture doc. 
No specific discussion was had on the actual issue, yet.

**Comment by aj-stein-nist** [comment]
> Hi, @aj-stein-nist We just moved this to SCRAPI to focus the discussion under this repo as we're splitting out the APIs to it's own draft, from the Architecture doc. No specific discussion was had on the actual issue, yet.

But the referenced issue we did have some questions particularly on the media type for errors, I linked to the open question regarding why we ought to do it from Orie. I do not think the current OpenAPI spec reflects changing the media type. So maybe not discussion from others, but the implied task in the issue that one contributor asked is why change the OAS spec and I was pointing to why (following another IETF RFC).

So do we need to discuss this on the mailing list?
